ws.py

```diff
@@ -86,6 +87,8 @@
 if not GlobalVars.metasmoke_ws_host:
     log('info', "No metasmoke websocket host found, which is okay if you're anti-websocket")
 
+# Register actually_add_queue_timings_data hook
+atexit.register(actually_add_queue_timings_data)
```

Comment:
`atexit` unfortunately isn't going to work here, because we exit Smokey with `os._exit` which bypasses `atexit` handlers (instead of using `sys.exit`). I'm not sure why we use that...I'd suggest changing it to `sys.exit`, if not for [Chesterton's fense](https://en.wikipedia.org/wiki/G._K._Chesterton#Chesterton's_fence).

We probably want to be sure to actually write the batches of data every ~100 entries or so, so that even if we crash we don't lose *too* much data.

Reply:
👍 If you're changing the exit call, be sure to change the one in [`metasmoke.py`](https://github.com/Charcoal-SE/SmokeDetector/blob/549ae6ba8582821cf4fa0a27b4aea23ed445ed1c/metasmoke.py#L230) too.